cmd/containers-storage/mount.go

```diff
@@ -22,7 +22,7 @@ func mount(flags *mflag.FlagSet, action string, m storage.Store, args []string)
 	moes := []mountPointOrError{}
 	for _, arg := range args {
 		if paramReadOnly {
-			result, err := m.MountImage(arg, []string{"noexec", "nosuid"}, paramMountLabel)
+			result, err := m.MountImage(arg, []string{}, paramMountLabel)
```

Comment:
… I don’t know. This behavior seems to be undocumented, looking at the commit history, I can’t find any discussion or rationale.

But then again, suddenly allowing things is risky. Do we know if anyone uses this command in production?

Adding an extra option to disable/override these defaults would be safer.

Reply:
This is still outstanding.

Reply:
We do not ship it in any package, I've always considered it only a debugging/testing tool. You can't stop people from using it.

We can argue this is relaxing the old behavior, but the old one seems wrong.  It was setting some flags that were not supposed to be there in any case.

Reply:
I can live with that. @nalind WDYT?

Reply:
I'd be fine with that - "noexec" is certainly a surprise.  I'm more surprised that whether we call `MountImage()` or `Mount()` depends on whether or not the read-only flag was passed in at the command line.  Anyway, a quick grep didn't turn up any use of this option in the tests, so there doesn't appear to be anything that depends on this.